src/extension.ts

```diff
@@ -39,9 +46,12 @@ export const deactivate = () => { }
 const decorationType = window.createTextEditorDecorationType({ after: { margin: '0 0 0 1rem' } });
 
 const processActiveFile = async (document: TextDocument) => {
-	const { enable } = workspace.getConfiguration('magic-json') as MagicJsonConfiguration;
+	const { enable, ignoreFiles } = workspace.getConfiguration('magic-json') as MagicJsonConfiguration;
+	const platform = process.platform;
+	const separator = platform === TypeOs.WINDOWS ? '\\' : '/';
+	const currentFilename = document.fileName.split(separator).slice(-1)[0];
 
-	if (!enable) {
+	if (!enable || (ignoreFiles && ignoreFiles.includes(currentFilename))) {
```

Comment:
question: should it support glob patterns? [micromatch](https://www.npmjs.com/package/micromatch) seems like a good library for this use case.

Reply:
Nope, that doesn't support glob pattern cause I don't need that. But yes, I can add micromatch.